diff --git a/node_modules/typeorm/commands/CommandUtils.js b/node_modules/typeorm/commands/CommandUtils.js
index 797456a..017ed01 100644
--- a/node_modules/typeorm/commands/CommandUtils.js
+++ b/node_modules/typeorm/commands/CommandUtils.js
@@ -26,7 +26,7 @@ class CommandUtils {
             throw new Error(`Given data source file must contain export of a DataSource instance`);
         }
         const dataSourceExports = [];
-        for (const fileExport of dataSourceFileExports) {
+        for (const fileExport of Object.values(dataSourceFileExports)) {
             // It is necessary to await here in case of the exported async value (Promise<DataSource>).
             // e.g. the DataSource is instantiated with an async factory in the source file
             const awaitedFileExport = fileExport instanceof Promise ? await fileExport : fileExport;
